core/controllers/blog_dashboard.py

```diff
@@ -353,3 +355,120 @@ def delete(self, blog_post_id: str) -> None:
         blog_domain.BlogPost.require_valid_blog_post_id(blog_post_id)
         blog_services.delete_blog_post(blog_post_id)
         self.render_json(self.values)
+
+
+class BlogDashboardBlogPostStatisticsHandler(
+    base.BaseHandler[Dict[str, str], Dict[str, str]]
+):
+    """Handler for blog dashboard statistics tab for blog post."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'blog_post_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'chart_type': {
+            'schema': {
+                'type': 'basestring',
+                'choices': [
+                    'views',
+                    'reads',
+                    'reading_time'
+                ]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS: Dict[str, Dict[str, str]] = {
+        'GET': {}
+    }
+
+    @acl_decorators.can_access_blog_dashboard
+    def get(self, blog_post_id: str, chart_type: str) -> None:
+        """Populates the data for generating statistics plot."""
+
+        if chart_type == 'views':
+            stats = blog_statistics_domain.BlogPostViewsAggregatedStats
```

Comment:
I'm not sure why a class is assigned to variable (stats) which is ultimately going to be assigned with different value. Can you please explain? 

And I think this is the main reason why MyPy checks are failing!!
https://github.com/oppia/oppia/actions/runs/3526837419/jobs/5915218501#step:7:129

Reply:
Removed!